Sources/http.swift

```diff
@@ -20,16 +20,17 @@ public class HTTP {
 }
 
   init(port: UInt16) {
-    serverSocket = socket(AF_INET, Int32(SOCK_STREAM.rawValue), 0)
+    serverSocket = socket(AF_INET, Int32(SOCK_STREAM), 0)
```

Comment:
I think we should make another branch for Mac and another branch for Linux :D your changes work well on Mac. But Swift for Linux has some different :D